song-server/src/main/java/org/icgc/dcc/song/server/service/ValidationService.java

```diff
@@ -67,30 +67,48 @@ public void asyncValidate(@NonNull String uploadId, @NonNull String payload, Str
 
   public void syncValidate(@NonNull String uploadId, @NonNull String payload, String analysisType) {
     log.info("Validating payload for upload Id=" + uploadId + "payload=" + payload);
+    log.info(format("Analysis type='%s'",analysisType));
+    val errors = validate(payload, analysisType);
+    update(uploadId,errors);
+  }
+
+  public String validate(@NonNull String payload, String analysisType) {
+    String errors;
+
     log.info(format("Analysis type='%s'",analysisType));
     try {
       val jsonNode = JsonUtils.readTree(payload);
 
-      if (analysisType == null) {
-        updateAsInvalid(uploadId, "Uploaded JSON document does not contain a valid analysis type");
+      if (jsonNode.has("study")) {
+        errors =  "Uploaded JSON document must not contain a study field";
+      } else if (analysisType == null) {
```

Comment:
this doesnt hurt to have, however the JsonSchema  can make this mandatory  and only of type `string` (i.e null make create a validation error). No action needed here, as in my next PR i tightened the json schema requirements.